coremltools/converters/mil/backend/mil/passes/test_passes.py

```diff
@@ -1141,3 +1141,47 @@ def program(x):
             backend=("mlprogram", "fp32"),
             expected_output_shapes={block.outputs[0].name: tuple(x_shape[:-1])},
         )
+
+    @pytest.mark.skipif(ct.utils._macos_version() < (12, 0), reason="mlprogram predict available only on macOS12+")
```

Comment:
This is an awesome fix!
@kasper0406 would you mind rebasing this PR on the tot branch?
Since we just have `coremltools` 8.1 released.

Reply:
Done